gst/nnstreamer/nnstreamer_plugin_api_util_impl.c

```diff
@@ -578,16 +660,21 @@ gst_tensors_info_parse_names_string (GstTensorsInfo * info,
     str_names = g_strsplit (name_string, ",", -1);
     num_names = g_strv_length (str_names);
 
```

Comment:
How about using a variable to sum of NNS_TENSOR_SIZE_LIMIT and NNS_TENSOR_SIZE_EXTRA_LIMIT ?